packages/relayer/indexer/indexer.go

```diff
@@ -299,13 +299,6 @@ func (i *Indexer) eventLoop(ctx context.Context, startBlockID uint64) {
 
 // filter is the main function run by Start in the indexer
 func (i *Indexer) filter(ctx context.Context) error {
-	n, err := i.eventRepo.FindLatestBlockID(i.eventName, i.srcChainId.Uint64(), i.destChainId.Uint64())
-	if err != nil {
-		return err
-	}
-
-	i.latestIndexedBlockNumber = n
```

Comment:
This part of code is unnecessary. It will overwrite https://github.com/taikoxyz/taiko-mono/blob/main/packages/relayer/indexer/set_initial_Indexing_block_by_mode.go#L13 while it should not.